commands/create/api-sample.ts

```diff
@@ -39,23 +39,31 @@ module.exports = {
       }
     }
 
-    const samplesConfig = await fetchFileFromRepository(
```

Comment:
This change probably feels irrelevant, but this was the only other usage of `fetchFileFromRepository` in the repo so I figured it would be worthwhile to remove it. The debug logs are significantly better if you directly call `fetchRepoFile`